src/lib/command-helpers.mjs

```diff
@@ -1,20 +1,73 @@
 //@ts-check
 
+import { container } from "../cli.mjs";
+import { getAccountKey, getDBKey, refreshDBKey } from "./auth/authNZ.mjs";
+
+// TODO: update for yargs
 function buildHeaders() {
   const headers = {
     "X-Fauna-Source": "Fauna Shell",
   };
-  if (!["ShellCommand", "EvalCommand"].includes(constructor.name)) {
-    headers["x-fauna-shell-builtin"] = "true";
-  }
+  // if (!["ShellCommand", "EvalCommand"].includes(constructor.name)) {
+  //   headers["x-fauna-shell-builtin"] = "true";
+  // }
   return headers;
 }
 
 export async function getSimpleClient(argv) {
+  const logger = container.resolve("logger");
+  const { profile, database: path, role, secret } = argv;
+  const accountKey = getAccountKey(profile).accountKey;
+  if (secret) {
+    logger.debug("Using Database secret from command line flag");
```

Comment:
the loggers besides stdout and stderr accept a second argument, `component`, which is used to filter verbose output. could you add a component for this (`client` or `network` maybe?). it'll need to be added to these debug calls and to the list of components in `cli.mjs`.